src/Provider/PaymentTokenProvider.php

```diff
@@ -15,26 +21,34 @@ final class PaymentTokenProvider implements PaymentTokenProviderInterface
     public function __construct(
         private readonly RepositoryInterface $orderRepository,
         private readonly RepositoryInterface $paymentTokenRepository,
-    ) {}
+    ) {
+    }
 
     public function provideToken(Request $request): ?PaymentSecurityTokenInterface
     {
+        /** @var string $content */
         $content = $request->getContent();
         $content = json_decode($content, true);
 
         $transactionData = $content['transaction'];
 
```

Comment:
How it would behave with this?

```suggestion

/** @var OrderInterface $order */
```